main.js

```diff
@@ -8,21 +8,40 @@ import Session from './Session.js';
 
 export {default as SessionService} from './SessionService.js';
 
+/**
+ * Creates a Session with an optional id or store.
+ *
+ * @param {object} options - Options to use.
+ * @param {string} [options.id = 'session.default'] - An id for the session.
```

Comment:
```suggestion
 * @param {string} [options.id='session.default'] - An id for the session.
```
The style in the rest of the DB code has no spaces around the equal sign for JSDoc default values.

Reply:
oh my you are correct. that is also not just our style, but actual jsdoc format =) I will correct it in a second.